README.md

```diff
@@ -14,52 +14,57 @@ git clone https://github.com/ComputationalRadiationPhysics/openPMD-converter-GDF
 ```
 or download an archive from the [github page](https://github.com/ComputationalRadiationPhysics/openPMD-converter-GDF) and unpack it.
 
-The only dependency is [h5py](https://www.h5py.org/), which can be installed e.g., by
+The only dependency is [openpmd_api](https://github.com/openPMD/openPMD-api), which can be installed e.g., by
 ```
-pip install h5py
+python -m pip install openpmd-api
 ```
 
 ## Converting from GDF to openPMD
 
-To convert a GDF file into an openPMD-conforming hdf5 file run the ```gdf_to_hdf.py``` module as follows:
+To convert a GDF file into an openPMD-format file run the ```gdf_to_openPMD.py``` module as follows:
 ```
-python gdf_to_hdf gdf_file [hdf5_file]
+python gdf_to_openPMD -gdf_file gdf_file -openPMD_output
 ```
 where 
 * ```gdf_file``` is the path to an input GDF file;
-* ```hdf5_file``` is the path to an output openPMD-conforming hdf5 file, by default ```result.h5```.
+* ```file in openPMD-format``` is the path to an output openPMD-format file.
+
+The format is selected according to the file extension: current supported: .h5 (HDF5) .bp (ADIOS1) or .json (JSON).
```

Comment:
`current` -> `currently`.
I think `.h5` and such are better formatted as code. Hyperlinks would also be nice.